e2e/test/iothub/NoRetryE2ETests.cs

```diff
@@ -87,8 +87,8 @@ public async Task DuplicateDevice_NoRetry_NoPingpong_OpenAsync()
 
             Logger.Trace($"{nameof(DuplicateDevice_NoRetry_NoPingpong_OpenAsync)}: 2 device client instances with the same deviceId={testDevice.Id}.");
 
-            using DeviceClient deviceClient1 = testDevice.CreateDeviceClient(Client.TransportType.Amqp_Tcp_Only);
-            using DeviceClient deviceClient2 = testDevice.CreateDeviceClient(Client.TransportType.Amqp_Tcp_Only);
+            using DeviceClient deviceClient1 = testDevice.CreateDeviceClient();
```

Comment:
If this is omitted because amqp is the default transport type (right now), I think specifying the transport type in our E2E tests explicitly will help with readability. What do you think?

Reply:
I meant to replace all instances where it was specified with a client options. I just missed this one. Thanks for catching it.